wremnants/include/utils.h

```diff
@@ -40,10 +40,25 @@ float mt_wlike_nano(float pt, float phi, float ptOther, float phiOther, float me
   met_wlike.SetMagPhi(met,phimet);
   met_wlike = pl + met_wlike;
 
-  return std::sqrt(2*pt*met_wlike.Mod()*(1-std::cos(phi-met_wlike.Phi())));
+  return met_wlike;
+  
+}
+
+    
+float get_mt_wlike(float pt, float phi, float ptOther, float phiOther, float met, float phimet) {
+
+    TVector2 met_wlike = get_met_wlike(ptOther, phiOther, met, phimet);
+    return std::sqrt(2*pt*met_wlike.Mod()*(1-std::cos(phi-met_wlike.Phi())));
```

Comment:
met_wlike can be made const
```const TVector2 met_wlike = get_met_wlike(ptOther, phiOther, met, phimet);```
The mT equation is written out three times in this file, here, above (mt_2), and below (get_mt_wlike). In this case we can reuse the function below:
``` return get_mt_wlike(pt, phi, met_wlike)```

Reply:
Good point, I am working on it